### PR TITLE
Fixed the Android unit test failures

### PR DESCRIPTION
**Pull request by prashantvc**
Fixed the unit test failurs 
### PR Checklist ###

- [X] Has tests (if omitted, state reason in description)
- [X] Rebased on top of master at time of PR
- [X] Consolidate commits as makes sense

### CODE REVIEW

**Comment by prashantvc** [comment]
It's strange Github including old commit in the PR. Anyway the new commits starts from `6224dc9`

File: src/Plugin.Settings.Shared/CrossSettings.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@
  */
 
 using System;
+using System.Diagnostics;
```
Comment (jamesmontemagno) [inline src/Plugin.Settings.Shared/CrossSettings.cs]: Doesn't look like this needs to be checked in...

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]

File: src/Plugin.Settings.Shared/CrossSettings.cs (hunk 1 above)
Comment (prashantvc) [reply src/Plugin.Settings.Shared/CrossSettings.cs]: yeah, it slipped between cracks. I will update it :)

Comment (prashantvc) [reply src/Plugin.Settings.Shared/CrossSettings.cs]: Fixed this, read for merge

**Comment by prashantvc** [comment]
Fixed the review findings, reverted _CrossSettings.cs_ file back to commit cb2fe5e

**Review by jamesmontemagno — APPROVED** [review]

**Pull request merged**